Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdater.cs

```diff
@@ -100,11 +100,15 @@ AppContentSummary latestVersionContentSummary
             if (repairCost < contentSize)
```

Comment:
repairCost seems to be a number of missing and broken files (x2). Is contentSize the content size in bytes?
This does not seem right.

Reply:
Below that there is:
```csharp
if (isNewVersionAvailable)
{
    repairCost *= latestVersionContentSummary.Chunks.Size;
}
else
{
    repairCost *= installedVersionContentSummary.Chunks.Size;
}
```

repairCost is estimated to be equal to the sum of missing and broken files, multiplied by two and then multiplied by chunk size in the respective version.

Reply:
Sorry, I didn't notice that.
I can't imagine it though. For instance if there's a large file broken, would the cost be the same as for small file?

Reply:
I changed it now. The cost calculation will now use actual file sizes.